testData/solutions/web.json

```diff
@@ -602,7 +603,8 @@
                             }
                         }
                     },
-                    "pictogramsEnabled": "http://registry\\.gpii\\.net/common/pictogramsEnabled"
+                    "pictogramsEnabled": "http://registry\\.gpii\\.net/common/pictogramsEnabled",
```

Comment:
Hi Kasper,

I should have mentioned that pictogramsEnabled was part of a previous pull request and already has a transformer: https://github.com/GPII/universal/blob/master/testData/solutions/web.json#L605.